hudi-client/src/test/java/org/apache/hudi/table/TestMergeOnReadTable.java

```diff
@@ -968,6 +968,7 @@ public void testInsertsGeneratedIntoLogFilesRollback() throws Exception {
       Thread.sleep(1000);
       // Rollback again to pretend the first rollback failed partially. This should not error our
       writeClient.rollback(newCommitTime);
+      folder.delete();
```

Comment:
does nt this happen already at the HoodieClientTestHarness level?

Reply:
Won't since it is a variable in the method.